test/manual/api/improved_api_tests.sh

```
@@ -0,0 +1,676 @@
+  fi
+}
+
+# FUNCTION: Delete connections and systems
+delete_everything() {
+  echo "==== Deleting Connections and Systems ===="
+  load_connections
+  load_systems
+
+  echo "Cleaning up connections..."
+  # Delete connections using source/target pairs
+  local conn_array=($CREATED_CONNECTION_IDS)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)